tests/Kernel/ParagraphsTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Drupal\Tests\oe_theme\Kernel;
+
+use Drupal\paragraphs\Entity\Paragraph;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Class ParagraphsTests.
+ *
+ * @package Drupal\Tests\oe_theme\Kernel
+ */
+class ParagraphsTest extends AbstractKernelTest {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'paragraphs',
+    'user',
+    'system',
+    'file',
+    'field',
+    'entity_reference_revisions',
+    'link',
+    'text',
+    'oe_paragraphs',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('paragraph');
+    $this->installSchema('system', ['sequences']);
+    $this->installConfig(['oe_paragraphs']);
+  }
+
+  /**
+   * Test links block paragraph rendering.
+   */
+  public function testLinksBlock() {
+    $paragraph = Paragraph::create(['type' => 'oe_paragraphs_links_block']);
+    $paragraph->field_oe_paragraphs_text = 'Title';
```

Comment:
To improve readability, we can move these field values inside the array specified in the `create()` method above.